scripts/globals/item_utils.lua

```diff
@@ -0,0 +1,41 @@
+require("scripts/globals/status")
+require("scripts/globals/msg")
+
+item_utils = {}
+
+function item_utils.skillBookCheck(target, skillID)
+    local skill = skillID
+    local mainCap = target:getMaxSkillLevel(target:getMainLvl(), target:getMainJob(), skill) or 0
+    local subCap = target:getMaxSkillLevel(target:getSubLvl(), target:getSubJob(), skill) or 0
+    local skillLevel = target:getCharSkillLevel(skill)/10
+    local mainDif = (mainCap*10)/10 - (target:getCharSkillLevel(skill)*10)/100
+    local subDif = (mainCap*10)/10 - (target:getCharSkillLevel(skill)*10)/100
+    local noSkill = 0
+    local canUse = 0
+
+    if mainCap == 0 then
+        noSkill = noSkill +1
```

Comment:
Can you please add a space after +?  (repeats below)

Reply:
all done done